src/test/ui/const-generics/infer/uninferred-consts.min.stderr

```diff
@@ -2,7 +2,12 @@ error[E0282]: type annotations needed
   --> $DIR/uninferred-consts.rs:14:9
    |
 LL |     Foo.foo();
-   |         ^^^ cannot infer the value of const parameter `N` declared on the associated function `foo`
+   |         ^^^ cannot infer the value of const parameter `A` declared on the associated function `foo`
+   |
+help: consider specifying the const argument
+   |
+LL |     Foo.foo::<A>();
```

Comment:
Isn't this help incorrect?

Reply:
also, what happens if we can infer the first param but not the second one?

For example with
```rust
#![feature(min_const_generics)]

fn foo<const N: usize, const M: usize>() -> [u8; N] {
    todo!()
}

fn main() {
    let _: [u8; 17] = foo();
}
```

Reply:
It will just show the second parameter in the help message

Reply:
Because `Foo.foo::<13>()` doesn't compile, it requires `Foo.foo::<13, 17>()`. You can't only show mention some of the generic params. It's either none or all of them.

Can you add the above example as a test, potentially with a `FIXME(const_generics)`?

Reply:
Yea because we don't have access to the list of all uninferred parameters in the context.

Reply:
I think this is already covered in the other `FIXME`. Do we really need another fixme?

Reply:
would be better imo, just in case fixing the first one does not actually fix the second one :thinking: